### ISSUE TITLE
output isn't readable in some cases

### ISSUE THREAD

**Issue by ivnovi** [bundyfx/vamp#19]
it would be nice to control amount of output you get when running prep/apply and limit to only minimal information unless you want to debug specific config. some resources generate lots of usually unnecessary output, for example cChocoInstaller

**Comment by flynnCoolblue** [comment]
Yep, Maybe verbose should not be by default but rather passed in by the user if required?
